moj_analytics/auth0_client.py

```diff
@@ -116,7 +120,7 @@ def create(self, resource):
 
         response = self.request('POST', endpoint, json=resource)
 
-        if 'error' in response:
+        if 'error' in response or response == {}:
```

Comment:
I didn't know `requests`' response could be an empty dictionary, is that normal? Shouldn't always have at least the status code and other things in it even on error?

Reply:
That's the point. The `response` could be an empty dictionary returned by `self.request` (see https://github.com/ministryofjustice/analytics-platform-concourse-auth0-client-resource/blob/877062bec0b1d6a011de659570503d24e255ea4f/moj_analytics/auth0_client.py#L116)

Reply:
Oh I see, I thought the `response` variable was coming from `requests` directly but it's actually coming from one of our functions, that's why it could be `{}`, thanks for clarifying that 👌